website/pages/vmware/downloads/index.jsx

```diff
@@ -1,42 +1,79 @@
 import s from './style.module.css'
-import { VMWARE_UTILITY_VERSION } from 'data/version.json'
-import ProductDownloader from '@hashicorp/react-product-downloader'
+import { VMWARE_UTILITY_VERSION } from 'data/version'
+import ProductDownloadsPage from '@hashicorp/react-product-downloads-page'
+import { generateStaticProps } from '@hashicorp/react-product-downloads-page/server'
 import Head from 'next/head'
 import HashiHead from '@hashicorp/react-head'
 
-export default function DownloadsPage({ releaseData }) {
+export default function DownloadsPage(staticProps) {
   return (
-    <div className={s.root}>
+    <>
+      <ProductDownloadsPage
+        pageTitle="Download Vagrant vmware Utility"
+        getStartedDescription="Follow step-by-step tutorials on the essentials of Vagrant VMWare Utility."
+        getStartedLinks={[
+          {
+            label: 'Installation Instructions',
+            href:
+              'https://www.vagrantup.com/docs/providers/vmware/installation',
+          },
+          {
+            label: 'Community Resources',
+            href: 'https://www.vagrantup.com/community',
+          },
+          {
+            label: 'View all Vagrant tutorials',
+            href: 'https://learn.hashicorp.com/vagrant',
+          },
+        ]}
+        logo={<p className={s.notALogo}>Vagrant vmware Utility</p>}
+        tutorialLink={{
+          href: 'https://learn.hashicorp.com/vagrant',
+          label: 'View Tutorials at HashiCorp Learn',
+        }}
+        packageManagerOverrides={[
+          // Note: Duplicate Homebrew entries target
+          // both macOS and Linux. If one is removed,
+          // then Homebrew will show up under the Linux tab.
+          {
+            label: 'Homebrew',
+            os: 'NONE--IGNORE',
+          },
+          {
+            label: 'Homebrew',
+            os: 'NONE--IGNORE',
+          },
+          {
+            label: 'Amazon Linux',
+            os: 'NONE--IGNORE',
+          },
+          {
+            label: 'Fedora',
+            os: 'NONE--IGNORE',
+          },
+          {
+            label: 'Ubuntu/Debian',
+            os: 'NONE--IGNORE',
+          },
+          {
+            label: 'CentOS/RHEL',
+            os: 'NONE--IGNORE',
```

Comment:
What's going on with this?

Reply:
This was the only way I could find to stop the package managers from showing up. Open to other ways to configure just the binary downloads to show up, if there is one 🙇

Reply:
Added a comment here to clarify the `NONE--IGNORE` part - definitely open to iterating on this if we find a better way.

Reply:
Update here based on recent sync: @jescalan is looking at making a change to allow us to not do this, as it's certainly an unexpected use of the interface. If feasible, we might update the ProductDownloadsPage component so that "package managers" can be disabled entirely (that's the use case here, as vagrant-vmware-utility support in package managers is unknown). Alternately, if not feasible, we'll proceed with this pattern (for now).